releasenotes/notes/add-variational-quantum-time-evolution-112ffeaf62782fea.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add algorithms for Variational Quantum Time Evolution that implement a new interface for
+    Qantum Time Evolution. The feature supports real (:class:`qiskit.algorithms.VarQRTE`.) and
+    imaginary (:class:`qiskit.algorithms.VarQITE`.) quantum time evolution according to a
+    variational principle provided.
+
```

Comment:
I imagine you'll expand on this somewhat, There are 47 files in this PR  - yes some are init files and test etc but there are way more than the 2 main algorithms that I would have thought might be documented here. And I imagine you had planned to do some code samples here.

Reply:
The main algorithms are actually only VarQITE and VarQRTE. The rest are just algorithms needed for the implementation of the former 2. We do not expect auxiliary algorithms to be used by the users directly, in separation. I might add information about these auxiliary algorithms but I thought that in release notes it is best to focus on the user-facing ones... Let me know what you think.

Reply:
I think we need code samples here at least. If say we expose calculators to a user then we can at least link to the place where they are documented and give a brief statement that has an overview of their purpose as part of some configuration. It just seems we have a lot of things here with perhaps choices that a user can make when setting up the above that could at least be outlined here. For the amount of work here the text does not seem to really do it justice I think.

Reply:
It would be great to add a snippet on how to use the new feature here, this could just be a copy paste from the docs you've written I think 🙂

Reply:
I extended text as well as added a code example.